docs/docusaurus/docs/integrations/integration_datahub.md

```diff
@@ -8,106 +8,14 @@ sidebar_label: "DataHub"
 sidebar_custom_props: { icon: 'img/integrations/datahub_icon.png' }
 ---
 
-:::info
-* Maintained By: DataHub
-* Status: Beta
-* Support/Contact: https://slack.datahubproject.io/
-:::
+DataHub is a modern data catalog built to enable end-to-end data discovery, observability, and governance.  DataHub is built on an extensible metadata platform that helps manage the complexity of diverse and evolving data ecosystems.
```

Comment:
```suggestion
DataHub is a modern data catalog built to enable end-to-end data discovery, observability, and governance.  DataHub is built on an extensible metadata platform that helps manage the complexity of diverse and evolving data ecosystems.
```

Just an aside - I'm not sure why we're explaining what the applications are and why they are helpful. I would suggest that all that is required is a link to the source documentation. Your call.